plugins/katello/3.7/release_notes/release_notes.md

```diff
@@ -4,71 +4,77 @@ title: Release Notes
 
-Performance improvements across the board have brought up to a 50% improvement in Content View Publishes, 95% improvement in Promotions, and a 95% improvement in Composite Content View publishes.
+### New Red Hat Repositories Page
+Completely redesigned with an eye toward usability and functionality.
 
-## Deprecation Warnings
+### New Upstream Subscriptions API
+For users with Red Hat content we now provide an API for managing subscriptions against existing manifests. Check the API documentation for details.
 
-#### File Repository Paths on Upgrade
+### Content Views
+Composite content views can be configured to auto-publish when one of its components is changed.
 
-Up until now file repositories were not able to be synced out to a proxy due to the URL path that the published file repositories were generated at. Katello 3.6 modifies the published path to mirror paths similar to how yum and puppet modules are published. This allows proper HTTPS serving of files repositories and syncing them out to proxy along with a more obvious and readable path. The downside of this needed change is that all file repositories are republished to a new path rendering all previous paths incorrect for any changes to the file repositories from the point of upgrade forward. The original file repository paths are left in tact, snapshotted at the point of the upgrade. This will allow a small window to update any infrastructure that relies on those file paths.
+### Client Tooling
+Zypper and DNF plugins have been added to support errata applicability similar to Yum clients
 
-Old file repositories can be cleaned up at a user's discretion to reclaim disk space by running:
+### Backup and Restore Utilities
+We are beginning to adopt [Foreman Maintain](https://theforeman.org/plugins/foreman_maintain/0.2/index.html) for Katello. Our first endeavor has been to utilize it for backup and restore capabilities.
 
-```bash
-foreman-rake katello:cleanup_old_file_repos
-```
+The primary goals of this change are to:
+* Centralize system operations in a single tool
+* Support remote databases
+* Support new Mongo versions (3.4)
 
-## Contributors
- - Adam Ruzicka
- - Andrew Kofink
- - Brad Buckingham
- - Bryan Kearney
- - Chris Duryee
- - Christine Fouant
- - Dan Seethaler
- - Daniel Lobato Garcia
- - Daniel Lobato García
- - Eric D. Helms
- - Evgeni Golov
- - Ewoud Kohl van Wijngaarden
- - Himanshu
- - Ivan Nečas
- - John Mitsch
- - Jonathon Turel
- - Justin Sherrill
- - Klaas Demter
- - Lukas Zapletal
- - Marek Hulan
- - Masatake YAMATO
- - Matthias Dellweg
- - Michael Moll
- - Nagoor Shaik
- - Ondrej Prazak
- - Partha Aji
- - Paul Donohue
- - Ranjan Kumar
- - Sean O\'Keeffe
- - Sebastian Gräßl
- - Shimon Shtein
- - Shira Maximov
- - Stephen Benjamin
- - Tom McKay
- - Tomas Strachota
- - Tomer Brisker
- - Walden Raines
- - Zjhuntin
- - aagrawal
- - kgaikwad
+The old tools are completely removed so please be sure to update any cron jobs or scripts using them.
```

Comment:
Can you be specific about `old tools` and say `katello-backup and katello-restore`?

Reply:
Also could we put this in bold? Its a pretty important point so want to make sure users don't miss it